src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -29,14 +29,59 @@ import { makeTestGroup } from '../../../common/framework/test_group.js';
 import {
   kAllTextureFormats,
   kAllTextureFormatInfo,
-  kUncompressedTextureFormats,
   kTextureUsages,
 } from '../../capability_info.js';
 import { GPUConst } from '../../constants.js';
 import { maxMipLevelCount } from '../../util/texture/base.js';
 
 import { ValidationTest } from './validation_test.js';
 
+const kValidTextureFormatsForMSAA = [
```

Comment:
If we can't simply used "Uncompressed" then IMO we should add this info to the format capability tables (as a new column, `multisample: boolean`). Then instead of `kValidTextureFormatsForMSAA.includes(fmt)`, `kAllTextureFormatInfo[fmt].multisample`

Reply:
Done.